.github/workflows/secret-check.yml

```diff
@@ -0,0 +1,11 @@
+name: Secrets Check - Commit
+
+on: [push]
+
+jobs:
+  security-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
```

Comment:
I am confused by this syntax. It seems like this action would checkout `master` but it does appear to checkout the `actions-secret-check` in the logs

Reply:
That is curious. I believe it will change to `develop` after merge and then eventually `master`. Will have to double check each stage to make sure.